gapis/api/gles/replay.go

```diff
@@ -331,6 +352,25 @@ func (a API) QueryFramebufferAttachment(
 	return res.(*image.Data), nil
 }
 
+func (a API) SupportsPerfetto(ctx context.Context, i *device.Instance) bool {
```

Comment:
It seems like this should be a function of the device, rather than a function of the API.